webdriver-spec.html

```diff
@@ -742,6 +742,9 @@
 
   <p>Otherwise, let <var>parameters</var> be null.</p>
 
+ <li><p>If a <a>navigation</a> is currently underway <a>wait for
+  navigation to complete</a>.
```

Comment:
It applies to all commands. See: [nsCommandProcessor.js](https://github.com/SeleniumHQ/selenium/blob/master/javascript/firefox-driver/js/nsCommandProcessor.js#L528) This unconditionally wraps every incoming command to conform with this.